docs/source/_static/custom.css

```
@@ -0,0 +1,3 @@
+.section dl.py:not(:first-of-type) {
+    margin-top: 1.5em;
```

This suggestion can be ignored, I just like it more with even more space:

```suggestion
    margin-top: 2em;
```

---

I tried 2em first 😂